frontend/src/components/VSourcesTable.vue

```diff
@@ -167,37 +191,79 @@ export default defineComponent({
 @tailwind utilities;
 
 @layer components {
-  .table {
-    @apply rounded-sm border-0 border-dark-charcoal-20;
+  .source-table {
+    @apply hidden rounded-sm border-0 border-dark-charcoal-20 md:table;
```

Comment:
Is there a difference between `hidden md:table` and `sm:hidden`? Wondering mostly because `sm:max-md:hidden` avoids encoding an implementation detail about the template in the style, while also reducing the overhead of understanding when this is visible, in particular compared to the mobile table which is `md:hidden`.

Reply:
This is made this way as the normal table is hidden until you hit the md breakpoint, and the mobile table is visible until you hit the md breakpoint. If doesn't answer your question could you try to reword it? I'm trying to understand the question but I'm getting lost a bit as the classes don't really make sense to me